testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -500,6 +564,41 @@ impl BitcoinRegtestController {
         }
     }
 
+    /// wait until the ChainsCoordinator has processed sortitions up to the
+    ///   canonical chain tip, or has processed up to height_to_wait
+    pub fn resync(&self, height_to_wait: Option<u64>) -> BurnchainTip {
```

Comment:
I was a bit confused by `resync`, should it be `pox_process_blocks` or something?

Reply:
I think it's more like "wait until sortitions processed" -- I could rename to `wait_for_sortitions`.

Reply:
Addressed in 98edb3cd9